src/bentoml/triton.py

```diff
@@ -450,16 +488,109 @@ class TritonServerHandle:
         default=None, converter=_to_mut_iterable
     )
 
+    def _validate_args(self, args: str, value: t.Any, timeout: int | float = 0.3):
+        # NOTE: If the arguments are not available, the process will exit instantly.
+        # Otherwise, if it exists, then quickly quit the process with a timeout.
+        from ._internal.container.base import Arguments
+
+        cmd = Arguments([self._binary])
+        cmd.construct_args(self.model_repository, opt="model-repository")
+        cmd.construct_args(value, opt=args)
+
+        try:
+            subprocess.run(
+                list(map(str, cmd)),
+                timeout=timeout,
+                check=True,
+                stdout=subprocess.DEVNULL,
+                stderr=subprocess.DEVNULL,
+            )
+        except subprocess.TimeoutExpired:
+            _logger.debug("Arguments %s are available in %s", args, self._binary)
+            return True
+        except subprocess.CalledProcessError:
+            _logger.debug("Arguments %s does not exists for %s", args, self._binary)
+            return False
+        except Exception as e:  # pylint: disable=broad-except
+            _logger.error("Caught exception while validating arguments: %s", args)
+            _logger.error(e)
+            raise
```

Comment:
Maybe we don't need to check upfront. If the Triton server does not support http or grpc, just let it fail with a clear error message.